concepticondata/conceptlists/Healey-1970-364.tsv

```diff
@@ -0,0 +1,365 @@
+Healey-1970-364-96	96	farm	201	FARM
+Healey-1970-364-97	97	fat (n)	323	FAT (ORGANIC SUBSTANCE)
+Healey-1970-364-98	98	fault	134	FAULT
+Healey-1970-364-99	99	feather	1201	FEATHER
+Healey-1970-364-100	100	fell, to	463	FELL (A TREE)
+Healey-1970-364-101	101	female	2105	FEMALE PERSON
+Healey-1970-364-102	102	fence	1690	FENCE
+Healey-1970-364-103	103	fetch	3551	FETCH
+Healey-1970-364-104	104	find, see	719	FIND
+Healey-1970-364-105	105	finger	1303	FINGER
+Healey-1970-364-106	106	little finger	2774	LITTLE FINGER
+Healey-1970-364-107	107	fire	221	FIRE
+Healey-1970-364-108	108	firewood	10	FIREWOOD
+Healey-1970-364-109	109	fish	227	FISH
+Healey-1970-364-110	110	flare		
+Healey-1970-364-111	111	flat	1633	FLAT
+Healey-1970-364-112	112	flea	232	FLEA
+Healey-1970-364-113	113	flesh (of body)	2259	FLESH
+Healey-1970-364-114	114	flower	239	FLOWER
+Healey-1970-364-115	115	fly, to	1441	FLY (MOVE THROUGH AIR)
+Healey-1970-364-116	116	flying fox	235	FLYING FOX
+Healey-1970-364-117	117	foot	1301	FOOT
+Healey-1970-364-118	118	four	1500	FOUR
+Healey-1970-364-119	119	frog	503	FROG
+Healey-1970-364-120	120	fruit	1507	FRUIT
+Healey-1970-364-121	121	garden	586	GARDEN
+Healey-1970-364-122	122	gather	7	GATHER
+Healey-1970-364-123	123	get up	570	STAND UP
+Healey-1970-364-124	124	ghost	1175	GHOST
+Healey-1970-364-125	125	girl	1646	GIRL
+Healey-1970-364-126	126	give	1447	GIVE
+Healey-1970-364-127	127	go away		
+Healey-1970-364-128	128	goes out, (fire)		
+Healey-1970-364-129	129	grandchild	1617	GRANDCHILD
+Healey-1970-364-130	130	grandparent	2399	GRANDPARENT
+Healey-1970-364-131	131	ground	2159	GROUND
+Healey-1970-364-132	132	guard	1940	GUARD
+Healey-1970-364-133	133	hair (of head, of body)	1040	HAIR
+Healey-1970-364-134	134	hand	1277	HAND
+Healey-1970-364-135	135	he	1211	HE
+Healey-1970-364-136	136	head	1256	HEAD
+Healey-1970-364-137	137	hear	1408	HEAR
+Healey-1970-364-138	138	heart	1223	HEART
+Healey-1970-364-139	139	heavy	1210	HEAVY
+Healey-1970-364-140	140	heel	980	HEEL
+Healey-1970-364-141	141	her	2092	HIS OR HER
+Healey-1970-364-142	142	hide (it)	602	HIDE (CONCEAL)
+Healey-1970-364-143	143	high	1265	HIGH
+Healey-1970-364-144	144	his	2092	HIS OR HER
+Healey-1970-364-145	145	hit	1433	HIT
+Healey-1970-364-146	146	hole	1667	HOLE
+Healey-1970-364-147	147	hot	1286	HOT
+Healey-1970-364-148	148	house	1252	HOUSE
+Healey-1970-364-149	149	hungry	1533	BE HUNGRY
+Healey-1970-364-150	150	I	1209	I
+Healey-1970-364-151	151	inside	1606	INSIDE
+Healey-1970-364-152	152	intestines	1991	INTESTINES
+Healey-1970-364-153	153	intransitivizer		
+Healey-1970-364-154	154	island	622	ISLAND
+Healey-1970-364-155	155	it	169	IT
+Healey-1970-364-156	156	its		
+Healey-1970-364-157	157	jump	1829	JUMP
+Healey-1970-364-158	158	kill	1417	KILL
+Healey-1970-364-159	159	knee	1371	KNEE
+Healey-1970-364-160	160	knife	1352	KNIFE
+Healey-1970-364-161	161	knife, bush		
+Healey-1970-364-162	162	knuckle	2824	KNUCKLE
+Healey-1970-364-163	163	ladder	1464	LADDER
+Healey-1970-364-164	164	lance		
```

Comment:
Perhaps we can map this as 945 SPEAR?

Reply:
I can't tell if this is a verb or a noun, which is why I left it alone